index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Homework1</title>
+  <link rel="stylesheet" href="styles.css" type="text/css">
+  </head>
+  <body>
+      <header>
+          <nav>
+              <ul>
+                  <li><a href="">Home</a></li>
+                  <li><a href="">About me</a></li>
+                  <li><a class="active" href="">Base page</a></li>
+              </ul>         
+          </nav>
+      </header>
+      <center>
+      <div class="main">
+          <h2>What do I miss about Munich now when in isolation?</h2>
+          <section class="card">
+              <h2 class="card_title">Nordbad</h2>
+              <div class="container"></div>
+              <div class="card_imgWrapper">
+                   <img class="card_img" src="nordbad.jpg">
+              </div>
+              <div class="card_description"><br>Nordbad is a fun and relaxing place where to go on a weekend </br>when there's bad weather and you find it <a href="https://www.google.com/maps/place/Nordbad/@48.1603281,11.5615484,17z/data=!3m1!4b1!4m5!3m4!1s0x479e75deb97b8337:0x6bca64d0f3009f64!8m2!3d48.1603245!4d11.5637424">here</a>
+              </div>
+          </section>
+
+          <section class="card">
+            <h2 class="card_title">Skiing nearby Munich</h2>
+            <div class="container"></div>
+            <div class="card_imgWrapper">
+                <img class="card_img" src="skiingnearmunich.jpg">
+            </div>
+            <div class="card_description"><br>Brauneck or any other nearby skiing places are great </br>for fun day-trips full of fresh air and sports. Get a train or drive a car <a href="https://www.google.com/maps/place/Ski-+und+Wandergebiet+Brauneck/@47.6636382,11.4510302,12z/data=!4m13!1m7!3m6!1s0x479d97800959c6d9:0x1d1e48aecf891b10!2sBrauneck!3b1!8m2!3d47.6636111!4d11.5211111!3m4!1s0x479d90a53dbec0d5:0x541d092224075f!8m2!3d47.6748663!4d11.5545273">here</a>
+            </div>
+        </section>
+        
+        <section class="card">
+            <h2 class="card_title">Fit One Gym</h2>
```

Comment:
I see a lot of classes that are not being used in your css file. Class attributes on html elements should only be added if they have a purpose. I would remove the ones you don't have css rules for to make things cleaner.

Reply:
True, thanks. Will edit.